test/services/sourceControl.spec.js

```diff
@@ -76,42 +76,43 @@ describe('services', () => {
       it('should convert a timestamp if an offset is provided', () => {
         const service = new SourceControlService();
         expect(
-          service.convertTimestamp({ timestamp: 1234567890, timezoneOffset: 360 })
+          service.convertTimestamp({ timestamp: 1234567890, timezoneOffset: 360 }),
         ).toStrictEqual(new Date((1234567890 - 360 * 60) * 1000));
       });
 
       it('should keep the timestamp if the offset is 0', () => {
         const service = new SourceControlService();
         expect(
-          service.convertTimestamp({ timestamp: 1234567890, timezoneOffset: 0 })
+          service.convertTimestamp({ timestamp: 1234567890, timezoneOffset: 0 }),
         ).toStrictEqual(new Date(1234567890000));
       });
     });
 
     describe('getFileHistory', () => {
-      it('should return null for empty project paths', () => {
+      it('should return null for empty project paths', async () => {
         const service = new SourceControlService();
-        expect(service.getFileHistory(null, 'test.txt')).resolves.toBeNull();
+        await expect(service.getFileHistory(null, 'test.txt')).resolves.toBeNull();
         return expect(service.getFileHistory(undefined, 'test.txt')).resolves.toBeNull();
       });
 
-      it('should return null for empty file paths', () => {
+      it('should return null for empty file paths', async () => {
         const service = new SourceControlService();
-        expect(service.getFileHistory('/test/path', null)).resolves.toBeNull();
+        await expect(service.getFileHistory('/test/path', null)).resolves.toBeNull();
         return expect(service.getFileHistory('/test/path', undefined)).resolves.toBeNull();
       });
 
       it('should return an empty array for invalid file path', async () => {
         const service = new SourceControlService();
-        // eslint-disable-next-line prettier/prettier
-        return expect(service.getFileHistory('/Invalid/Path', 'DoesNotExist.txt')).resolves.toStrictEqual([]);
+        return expect(
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line above